lib/src/editor.dart

```diff
@@ -753,6 +761,19 @@ blockquote {
           });
         }
       }
+    } else if (message == 'onfocus') {
+      if(!Platform.isAndroid) {
```

Comment:
Why not on Android `unfocus` it?

Reply:
on Android when you add `FocusScope.of(context).unfocus();` HTML editor will lost focus too. When we remove that line of code, when the focus is on the HTML editor, the other fields will automatically remove the focus